### ISSUE TITLE
Add new "sc-seq" UKD dataset to catalog

### ISSUE THREAD

**Issue by tmheunis** [abcd-j/data-catalog#39]
This will need assistance from us to generate a file list for the data that they have on local storage

**Comment by jsheunis** [comment]
Waiting for communication from the authors before this can be worked on. No urgency at the moment.

**Comment by mslw** [comment]
In today's labmeeting we looked at five new [datasets tagged with bulk sequencing](https://data.abcd-j.de/?tab=datasets&keyword=bulk%20sequencing) - not sure if these are the sc-seq datasets, but two of these list NCBI search result page as "homepage" (e.g. [this one](https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA1012710)) and individual search results can be opened to find file download links.

The content tab does not link to file content. If we combined the two (with some effort no doubt), we would have a practical realization of the "generate catalog for published data" (see also an KBI written for [GEO database](https://knowledge-base.psychoinformatics.de/kbi/0007/index.html)), something we didn't really do in the SFB1451 catalog.
